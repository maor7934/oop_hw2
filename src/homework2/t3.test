# check empty graph order:


CreateGraph A

ListNodes A

#check empty children:

CreateGraph B
CreateNode onlyNode 10
AddNode B onlyNode
ListChildren B onlyNode







