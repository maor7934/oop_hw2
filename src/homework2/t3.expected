# check empty graph order:


created graph A

A contains:

#check empty children:

created graph B
created node onlyNode with cost 10
added node onlyNode to B
the children of onlyNode in B are:







